tests/test_pca.py

```diff
@@ -127,9 +125,7 @@ def pca_params(
                 else {"tsqr", "randomized"}
             )
         elif array_type in {sparse.csr_matrix, sparse.csc_matrix}:
-            svd_solver = (
-                {"lobpcg", "arpack"} if zero_center else {"arpack", "randomized"}
-            )
+            svd_solver = {"arpack"} if zero_center else {"arpack", "randomized"}
```

Comment:
Are we losing `lobpcg` here?

Reply:
Huh, looking at it, we never had a `lobpcg` test ok

Reply:
This test is (sometimes) the lobpcg test, so we had one. And we still have one, this will run the code (sometimes) with the lobpcg arg.

This change just means that lobpcg went from “we expect no warning” to “we expect a warning”

If we remove the warning, I’ll move it back in here.